sample.config.yaml

```diff
@@ -46,7 +46,8 @@ checks:
       max_aggregate_distance: 6
     PublisherStalledCheck:
       enable: true
```

Comment:
you could set the `enable` to false here actually to disable the check.

Reply:
well the reason for this is to test it for a couple of symbols first (since the less liquid ones are noisy), if we set it to false then all will be disabled

Reply:
Hmmm i think this is because if this be `false` then other checks should override it and say `enable: yes`. But that's clear now.